mesh/include/mesh.h

```diff
@@ -0,0 +1,8 @@
+#ifndef MESH_H
+#define MESH_H
+
+void *initialize_shared_memory(char* shared_memory_name, size_t size);
```

Comment:
This is the function used to initialize shared memory.
It will create a shared memory space of name shared_memory_name, with size size, that can be shared by multiple processes.